src/errorHandler.ts

```diff
@@ -47,6 +47,11 @@ export function registerDisposables(): Disposable[] {
                 toggleStatusBarItem(editor);
             }
         }),
+        window.onDidChangeActiveTextEditor(editor => {
```

Comment:
This logic should go into `toggleStatusBarItem(editor)`

Reply:
Updated.

Reply:
Updated 👍

Reply:
I didn't notice at all the handler's are same at first. 😅